.github/CODEOWNERS

```diff
@@ -1266,6 +1266,10 @@ x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts @elastic/kib
 # Kibana Platform Security
 /.github/codeql @elastic/kibana-security
 /.github/workflows/codeql.yml @elastic/kibana-security
+/config/serverless.yml @elastic/kibana-security
```

Comment:
__note:__ We may not necessarily own these configs in the long term, but at this stage, we'd like to ensure the feature overrides are utilized as intended.

Reply:
The config folder is already owned by Core. 

https://github.com/elastic/kibana/blob/807da63c617a8a9ad0b0d90e20a065fd499940fa/.github/CODEOWNERS#L1240

I think you might be overriding the ownership with your addition? Or is it additive?

Reply:
Hmm, I think you're right, good catch! That's what I see in the [documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners):

> If you want to match two or more code owners with the same pattern, all the code owners must be on the same line. **If the code owners are not on the same line, the pattern matches only the last mentioned code owner**.

I'll include both Core and Security for these files to make sure we both own them.

Reply:
Changed to 
```
# Core
/config/ @elastic/kibana-core
/config/serverless.yml @elastic/kibana-core @elastic/kibana-security
/config/serverless.es.yml @elastic/kibana-core @elastic/kibana-security
/config/serverless.oblt.yml @elastic/kibana-core @elastic/kibana-security
/config/serverless.security.yml @elastic/kibana-core @elastic/kibana-security
```